<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWidget</class>
 <widget class="QWidget" name="ExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Maps to Export</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBoxNormal">
      <property name="text">
       <string>Normal</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditNormalPostfix">
      <property name="placeholderText">
       <string>_n</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxSpecular">
      <property name="text">
       <string>Specular</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditSpecularPostfix">
      <property name="placeholderText">
       <string>_s</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxParallax">
      <property name="text">
       <string>Parallax</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditParallaxPostfix">
      <property name="placeholderText">
       <string>_p</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxOcclusion">
      <property name="text">
       <string>Occlusion</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditOcclusionPostFix">
      <property name="placeholderText">
       <string>_o</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBoxDifusse">
      <property name="text">
       <string>Difusse</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditDiffusePostFix">
      <property name="placeholderText">
       <string>_d</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxPreview">
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditPreviewPostfix">
      <property name="placeholderText">
       <string>_v</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
